platforms/jvm/scala/src/integTest/groovy/org/gradle/scala/compile/UpToDateScalaCompileIntegrationTest.groovy

```diff
@@ -144,7 +144,10 @@ class UpToDateScalaCompileIntegrationTest extends AbstractIntegrationSpec implem
 
             tasks.withType(ScalaCompile) {
                 scalaCompileOptions.with {
-                    additionalParameters = (additionalParameters ?: []) + "-target:8"
+                    if (!additionalParameters.isPresent()) {
+                        additionalParameters = []
+                    }
+                    additionalParameters.add("-target:8")
```

Comment:
:x: This doesn't sound right.

Whoever sets the parameter here shouldn't need to be concerned whether the list is absent or not. The user intention here is to add a parameter, and they should be able to do so with just calling `.add()`.

I'm not sure what we need to achieve this. Shall we simply initialize this particular property to `[]` by default? Should that be the default for all collection properties? What is the point of having an absent collection property?